### ISSUE TITLE
Stringent Regression Tests

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACEatoms.jl#4]
@davkovacs 

I think it would be great if we can produce a number of stringent regression tests (not part of the standard test set) where we download some existing training sets and ensure that future versions of the code can reproduce the accuracy and performance. This is not a short-term thing, but if as you say more people are starting to use ACE then we need to make sure that we really maintain the performance of the code across versions. 

My group will help of course, but I think you might be the best person to produce the initial tests?

**Comment by cortner** [comment]
@gabor1 are there public links to some of your favourite training sets?

**Comment by gabor1** [comment]
we are slowly collecting some on https://libatoms.github.io/GAP/data.html

-- Gábor

Gábor Csányi
Professor of Molecular Modelling
Engineering Laboratory, University of Cambridge
Pembroke College Cambridge

Pembroke College supports CARA. A Lifeline to Academics at Risk.  http://www.cara.ngo/

**Comment by davkovacs** [comment]
I will create some when trying to reproduce ACE1 results with ACE2.

**Comment by cortner** [comment]
@David - @andresrossb and I have started implementing performance regression tests in ACE.jl; there are some very nice packages that allow us to do this conveniently, it is worth a look. I highly recommend including this here as well. (for the performance anyhow. Haven't looked at how to do the same for model accuracy.)
